<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MobiiImagePanel</class>
 <widget class="QWidget" name="MobiiImagePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MobiiImagePanel</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="requestFiles">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Récupérer les images depuis mon téléphone</string>
       </property>
       <property name="icon">
        <iconset resource="mobii.qrc">
         <normaloff>:/buttons/Resources/sync.png</normaloff>:/buttons/Resources/sync.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTreeView" name="filesTree">
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="deleteFile">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="mobii.qrc">
               <normaloff>:/buttons/Resources/trash.png</normaloff>:/buttons/Resources/trash.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="nameFile">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>562</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"> border: 1px solid #008287;</string>
             </property>
             <property name="text">
              <string>Image Name</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="saveFile">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="mobii.qrc">
               <normaloff>:/buttons/Resources/filesave.png</normaloff>:/buttons/Resources/filesave.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="currentImg">
           <property name="minimumSize">
            <size>
             <width>640</width>
             <height>480</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>640</width>
             <height>480</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"> border: 1px solid #008287;</string>
           </property>
           <property name="text">
            <string>Current Image</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mobii.qrc"/>
 </resources>
 <connections/>
</ui>
